app/src/main/java/org/mozilla/fenix/library/bookmarks/viewholders/BookmarkFolderViewHolder.kt

```diff
@@ -39,7 +39,7 @@ class BookmarkFolderViewHolder(
         setSelectionListeners(item, selectionHolder)
 
         if (!item.inRoots()) {
-            setupMenu(item)
+            super.bind(item, mode)
```

Comment:
It feels a bit weird to me to conditionally call the super method here. I don't think I've seen that very often. It might be better to have an `updateMenu` method in the base viewholder class for this

Reply:
Especially if we wanted to start adding stuff to the super method that applies to all folders for whatever reason, this would have to be changed anyway. Basically I'd rather anyone making future changes to the super method not have to be aware that it may not always be called.